src/App.scss

```diff
@@ -22,17 +23,33 @@
 }
 
 .soon {
-  background-color: green;
+  background-color: #ff776e;
 }
 
 .kinda-soon {
-  background-color: yellow;
+  background-color: #fdab6d;
 }
 
 .not-soon {
-  background-color: red;
+  background-color: #fee0b3;
 }
 
 .inactive {
   background-color: lightgrey;
 }
+
+.navLink {
+  margin-left: 2em;
+}
+
+.navContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.navStyle {
+  display: flex;
+  align-items: center;
+  align-content: space-between;
+}
```

Comment:
I think it would be best practice to have a separate scss file for the navbar styles. we could have a folder called `Navbar` which would contain `Navbar.js` and `Navbar.scss`. This will create a better developer experience ✨ 😄

Reply:
Agreed again Meesha! Maybe you can help us with structuring since you've had more experience with production code